### PR TITLE
New package: applet-window-buttons-0.8.1

### PR DESCRIPTION
**Pull request by ale-saglia**
Hello, I create a template to build a KDE window buttons applet (https://github.com/psifidotos/applet-window-buttons).

### CODE REVIEW

**Comment by ale-saglia** [comment]
Seems to be an issue in Travis when compiled for ARM. Without QT5 can't find qmake and with can't find moc.
Similar issue in #24131

**Comment by ale-saglia** [comment]
Found problem for ARM build, plasma-framework has a $PlasmaQuick_INCLUDE_DIR directed to /usr/include/KF5 and not /usr/{ARCH}/include/KF5

**Comment by ale-saglia** [comment]
Solved by adding plasma-framework-devel as hostmakedepends

File: srcpkgs/applet-window-buttons/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'applet-window-buttons'
+pkgname=applet-window-buttons
+version=0.8.1
```
Comment (ericonr) [inline srcpkgs/applet-window-buttons/template:3]: 0.9.0 is out

File: srcpkgs/applet-window-buttons/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+revision=1
+build_style=cmake
+hostmakedepends="extra-cmake-modules gettext qt5-host-tools qt5-qmake
+kdeplasma-addons5 plasma-framework-devel"
```
Comment (ericonr) [inline srcpkgs/applet-window-buttons/template:7]:
```suggestion
hostmakedepends="extra-cmake-modules gettext qt5-host-tools qt5-qmake
 kdeplasma-addons5 plasma-framework-devel"
```

File: srcpkgs/applet-window-buttons/template [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+makedepends="plasma-framework-devel kdecoration-devel qt5-devel"
+depends="plasma-workspace"
```
Comment (ericonr) [inline srcpkgs/applet-window-buttons/template:9]: I don't like having plugins depending on the application they are a plugin for... Maybe remove the dependency?

**Pull request closed without merging**